packages/relay/src/lib/eth.ts

```diff
@@ -326,7 +327,8 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} estimateGas()`);
     if (!transaction || !transaction.data || transaction.data === '0x') {
```

Comment:
Can we add a comment up here to highlight this check implies this is a transfer transaction and not a contract function execution